lib/ProductOpener/Products.pm

```diff
@@ -110,9 +110,9 @@ sub make_sure_numbers_are_stored_as_numbers($) {
 			# fields like "salt", "salt_value"
 			# -> used internally, should not be used by apps
 			# store as numbers
-			elsif (looks_like_number($product_ref->{nutriments}{$field}))  {	
```

Comment:
Was this removed intentionaly?

Reply:
I don't think so. Actually, I don't remember editing this line in this PR. The only intentional change was regarding the `$robotoff_url`. The rest should only be changed because of `.editorconfig`. Let me check and fix this. 🙂

Reply:
Fixed. Thanks!